frb_dart/lib/src/basic.dart

```diff
@@ -178,3 +179,8 @@ class WireSyncReturnStruct extends ffi.Struct {
   @ffi.Uint8()
   external int success;
 }
+
+bool uint8ListToBool(Uint8List raw) {
```

Comment:
uber-nit: Well, shall we put it somewhere else, say, helpers.dart? Since it is not very "basic" to be in basic.dart